views/cypress/utils/selectors.js

```diff
@@ -1,9 +1,15 @@
 export default {
     deleteItem: '[data-context="instance"][data-action="deleteItem"]',
     deleteClass: '[data-context="class"][data-action="deleteItemClass"]',
+    newClass: '[data-context="resource"][data-action="subClass"]',
     addItem: '[data-context="resource"][data-action="instanciate"]',
     itemForm: 'form[action="/taoItems/Items/editItem"]',
     itemClassForm: 'form[action="/taoItems/Items/editClassLabel"]',
+    classOptions: '[action="/taoItems/Items/editItemClass"]',
+    editClass: 'ul[class="plain action-bar content-action-bar horizontal-action-bar"]',
```

Comment:
This selector corresponds to the whole blue panel with action buttons
Maybe it's better to add something like `#item-class-schema`?